app/models/shelter.rb

```diff
@@ -8,14 +8,13 @@ def pet_count
   end
 
   def average_rating
-    reviews.sum do |review|
-      review.rating
-    end / reviews.count
+    if self.reviews.size <= 0
+        return 0
+    end
+    self.reviews.average(:rating)
```

Comment:
Nice use of the average function here! This is much cleaner and easier. I need to do more research into Active record aggregate functions!